### PR TITLE
feat: add producers

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

**Review by emalfroy — APPROVED** [review]

File: build.fsx [hunk 1]
```diff
@@ -70,6 +71,7 @@ Target.create "Pack_Solution" (fun _ ->
     "ParcelRegistry.Api.CrabImport"
     "ParcelRegistry.Migrator.Parcel"
     "ParcelRegistry.Api.BackOffice.Abstractions"
+    "ParcelRegistry.Producer"
```
Comment (ArneD) [inline build.fsx:74]: Don't need to create nuget package for producer

File: src/ParcelRegistry.Infrastructure/paket.references [hunk 2]
```diff
@@ -18,3 +18,14 @@ Microsoft.Extensions.Configuration.Json
...
 SourceLink.Embed.AllSourceFiles
 SourceLink.Copy.PdbFiles
+
+Destructurama.JsonNet
+
+Serilog
+Serilog.Settings.Configuration
+Serilog.Extensions.Logging
+Serilog.Sinks.Console
+Serilog.Sinks.File
```
Comment (ArneD) [inline src/ParcelRegistry.Infrastructure/paket.references:28]: Don't need file

File: src/ParcelRegistry.Producer/paket.references [hunk 3]
```diff
@@ -0,0 +1,18 @@
+Be.Vlaanderen.Basisregisters.AggregateSource
+Be.Vlaanderen.Basisregisters.AggregateSource.SqlStreamStore
+Be.Vlaanderen.Basisregisters.Api
+Be.Vlaanderen.Basisregisters.EventHandling.Autofac
+Be.Vlaanderen.Basisregisters.DataDog.Tracing.Sql
+Be.Vlaanderen.Basisregisters.GrAr.Contracts
+Be.Vlaanderen.BasisRegisters.MessageHandling.Kafka.Simple
+Be.Vlaanderen.BasisRegisters.MessageHandling.Kafka.Producer
```
Comment (ArneD) [inline src/ParcelRegistry.Producer/paket.references:8]: Don't use simple

File: src/ParcelRegistry.Producer/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  "ConnectionStrings": {
+    "Events": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "ProducerProjections": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
+    "ProducerProjectionsAdmin": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.ParcelRegistry;Trusted_Connection=True;TrustServerCertificate=True;"
+  },
+
+  "BaseUrl": "https://api.staging-basisregisters.vlaanderen/",
+
+  "DataDog": {
+    "Enabled": false,
```
Comment (ArneD) [inline src/ParcelRegistry.Producer/appsettings.json:11]: true by default

**Pull request merged**